gtk/src/default/gtk-3.20/_apps.scss

```diff
@@ -416,6 +416,31 @@ terminal-window {
           &:not(:backdrop):hover { background-image: image($base_hover_color); }
         }
       }
+
+    }
+
+    .nemo-inactive-pane .view, .nemo-inactive-pane iconview {
+      // Shades the background of the inactive pane
+      background-color: $bg_color;
```

Comment:
shouldn't this be `$backdrop_bg_color`?

Reply:
Yeah, that makes sense, thanks! I've changed it and pushed the commit to this branch, but it's not appearing under the PR - do I need to manually update it somehow?

Reply:
Maybe you pushed it to the wrong branch?

Reply:
That's definitely the sort of thing I would do, but when I click on the link to the branch at the top of this page it says there are 3 commits. Then if I click on "View pull request" it takes me back here, where there are only 2. Very odd!

Reply:
Weird indeed. You can either let us fix it for you in a next PR, or propose a new PR altogether if you like

Reply:
Maybe it's related to the new UI and a UI bug here. Looked into your repo and saw the three commits.

Reply:
I did get a couple of 500 errors from GitHub yesterday, so maybe it's just a fluke. I'll try opening a new PR.